content/en/user-guide/web-application/resource-browser/index.md

```diff
@@ -5,60 +5,70 @@ description: >
     The Resource Browser allows you to view and manage your local AWS resources through the LocalStack Web Application.
 ---
 
+## Introduction
+
 The LocalStack Resource Browser allow you to view, manage, and deploy AWS resources locally while building & testing their cloud applications locally. It provides an internal, integrated experience, similar to the AWS Management Console, to manage the ephemeral resources locally in a LocalStack container on your local machine.
 
 <img src="resource-browser.png" alt="LocalStack Web Application's Resource Browsers outlining various local AWS services" title="Resource Browser" width="900" />
 
-{{< alert title="Warning" color="warning">}}
-- If you encounter a `Network Failure` error message while accessing the Resource Browser, it is likely that the LocalStack container is not running or the instance is not reachable at the endpoint specified in the instance bookmark.
-{{< /alert >}}
```

Comment:
Have we removed this? Feel like is helpful

Reply:
It's at the very bottom now